package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please delete this file.

Reply:
removed this file, added this file in .gitignore